osi_object.proto

```diff
@@ -512,6 +512,205 @@ message MovingObject
         TYPE_ANIMAL = 4;
     }
 
+    //
+    // \brief Additional pedestrian data in case the moving object is a human.
+    //
+    // This is an extension to the \c MovingObject with additional information 
+    // describing a pedestrian in more detail.
+    // 
+    message PedestrianAttributes {
+
+
+        // List of all skeleton points of the pedestrian
+        // 
+        // The number of skeleton points may vary, based on the detail level of 
+        // the pedestrian model used. For example, some simulators will not include 
+        // detailed data of the fingers or eyes of the pedestrian.
+        // 
+        repeated SkeletonPoint skeleton_data = 2;
```

Comment:
skeleton_data has the number 2. This can be changed to 1 in the next commit.

Reply:
Has been fixed.